packages/icon/Icon.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Icon = ({ name, size, ...rest }) => (
+  <i className={`icon icon-${name} ${size ? `icon-${size}` : ''}`} {...rest} />
```

Comment:
Should we merge in prop className? Are there any a11y attributes which would be good to default (maybe `aria-label={name}`)?

Reply:
Yeah we can merge those in. And then set the default aria label to `name` of the icon unless specified otherwise.